cafebabel/static/css/styles.css

```diff
@@ -1805,6 +1805,59 @@ textarea#body {
 
 /* Homepage */
 
+.home article:first-of-type {
+  grid-area: pick;
+}
+
+.home article:nth-of-type(2) {
+  grid-area: latest1;
+}
+
+.home article:nth-of-type(3) {
+  grid-area: latest2;
+}
+
+.home article:nth-of-type(4) {
+  grid-area: latest3;
+}
+
+.home article:nth-of-type(5) {
+  grid-area: latest4;
+}
+
+.home article:nth-of-type(6) {
+  grid-area: latest5;
+}
+
+.home article:nth-of-type(7) {
+  grid-area: latest6;
+}
+
+.home article:nth-of-type(8) {
+  grid-area: latest7;
+}
+
+.home article:nth-of-type(9) {
+  grid-area: latest8;
+}
+
+.home #content {
+  display: grid;
+  grid-gap: 2rem;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-areas:
+    'detail detail detail detail detail detail'
+    'pick pick pick pick pick pick'
+    'latest1 latest1 latest1 latest1 ....... .......'
```

Comment:
I wonder if it wouldn't be easier to have your 3rd row being:
```
  'latests latests latests latests . .'
```
I wonder if [`subgrid` and/or `display: contents`](https://gridbyexample.com/video/subgrid-display-contents/) would not help.
That should also remove the [css repetition](https://github.com/cafebabel/cafebabel.com/pull/245/files/fe92a469a6629c15a3004b15e7382674d4e774c4..5bbb38561a65ba3409e2419821057566f2884046#diff-6569206075db87419fc4b34e54f5eb4cR1812) and permit more flexibility in the count of articles to show.

note that I'm very ignorant about grid so don't read this as anything else than a shared thought for an open discussion.

Reply:
I did it with `grid-column`, may be even simplier